pkg/splunk/enterprise/configuration.go

```diff
@@ -67,6 +67,20 @@ var defaultReadinessProbe corev1.Probe = corev1.Probe{
 	},
 }
 
+var defaultStartupProbe corev1.Probe = corev1.Probe{
+	InitialDelaySeconds: startupProbeDefaultDelaySec,
+	TimeoutSeconds:      startupProbeTimeoutSec,
+	PeriodSeconds:       startupProbePeriodSec,
+	FailureThreshold:    startupProbeFailureThreshold,
+	ProbeHandler: corev1.ProbeHandler{
+		Exec: &corev1.ExecAction{
+			Command: []string{
+				GetProbeMountDirectory() + "/" + GetStartupScriptName(),
```

Comment:
so this does not require an initial shell command like `/bin/bash` or something like that? Just providing the name of the script would work?

Reply:
Yes this is exactly same way we execute liveness and readiness probe.